config.json

```diff
@@ -1072,7 +1087,7 @@
         "sequences",
         "strings"
       ],
-      "unlocked_by": "tournament",
+      "unlocked_by": "yacht",
```

Comment:
This change makes logical sense, but we need to keep this one unlocked by tournament instead of yacht, since yacht is not a core exercise.  All the unlocked_by items must reference only core exercises.

Reply:
Done, as requested.